config/initializers/load_cloud_factory_service_variables.rb

```diff
@@ -0,0 +1,2 @@
+require('vcap_parser')
```

Comment:
This was missing in the frontend PR. Should it be added there as well? 

Nitpick: the brackets make it look like JS ;)

Reply:
Turns out frontend autoloads everything in `lib` whereas API doesn't, so it is required. Removed the brackets though!